model.py

```diff
@@ -2,11 +2,11 @@
 import torch.nn as nn
 import torchvision
 import torchvision.transforms as transforms
-import matplotlib.pyplot as plt
+# import matplotlib.pyplot as plt
```

Comment:
Can you explain why both of these imports were commented out? I understand removing one of the dupes. Or maybe matplotlib is unused in this file, in which case we should delete both comments